src/Slices/ServiceCatalog/UI/ServiceItem.tsx

```diff
@@ -37,10 +38,15 @@ export const ServiceItem: React.FunctionComponent<Props> = ({ service }) => {
       <DataListItemRow>
         <DataListItemCells
           dataListCells={[
-            <DataListCell key="primary content">
-              <Title id={serviceKey} headingLevel="h2" size="xl">
-                {service.name}
-              </Title>
+            <StyledDataListCell key="primary content">
```

Comment:
so you now have a flex-collumn in a flex-row? (no idea which direction the Flex component goes)

Reply:
as above

> Please connect issue in Zenhub ;)

it is connected :D 
![Screenshot 2022-11-21 at 10 05 59](https://user-images.githubusercontent.com/113331659/203009681-bb10c1cc-049b-42da-98d2-85d0eb8bb4a2.png)

Reply:
to clarify, flex was used there to keep all data in centre as they were slightly off-centre and now there were two components that should be in line(row) and one that under, so i decided to have one flex wrapper that handles these in row, and one general to keep all things centred  vertically